packages/platforms/delta/arm64/tn48m-dn/tn4810m-dn/platform-config/r0/src/python/arm64_delta_tn4810m_dn_r0/__init__.py

```diff
@@ -37,8 +37,8 @@ def baseconfig(self):
         subprocess.call('echo "-2" > /sys/bus/i2c/devices/i2c-2/2-0074/idle_state', shell=True)
         subprocess.call('echo "-2" > /sys/bus/i2c/devices/i2c-2/2-0075/idle_state', shell=True)
 
-        # Insert prestera kernel module
-        self.insmod("prestera_sw.ko")
-        self.insmod("prestera_pci.ko")
+        # Insert Marvell prestera modules by only probing prestera_pci module
+        # Suggested by Taras Chornyi
```

Comment:
Please put that into the commit message.

Reply:
Done and thanks.